implementations/strawberry-graphql/server.py

```diff
@@ -68,14 +68,38 @@ def get_product_dimensions() -> Optional["ProductDimension"]:
 
 
 def get_product_created_by() -> Optional["User"]:
-    return User(email="support@apollographql.com", total_products_created="1991", name="Jane Smith")
+    return User(
+        email="support@apollographql.com",
+        name="Jane Smith",
+        total_products_created="1337",
+        years_of_employment="10",
+    )
 
 
 @strawberry.federation.type(extend=True, keys=["email"])
 class User:
-    email: strawberry.ID = strawberry.federation.field(external=True)
+    email: strawberry.ID = strawberry.federation.field(external=True, override="users")
     name: Optional[str] = strawberry.federation.field(override="users")
     total_products_created: Optional[int] = strawberry.federation.field(external=True)
+    # TODO: the camel casing will be fixed in the next release of Strawberry
+    years_of_employment: int = strawberry.federation.field(external=True)
+    average_products_created_per_year: Optional[int] = strawberry.federation.field(
```

Comment:
I've updated https://github.com/apollographql/apollo-federation-subgraph-compatibility/blob/main/CONTRIBUTORS.md with information how to calculate the field

```
average_products_created_per_year = if (total_products_created != null) {
 round(1.0f * total_products_created /  years_of_employment)
} else {
  null
}
```

Reply:
Should we add a test that checks for this too?

Reply:
Unsure how we would test for that -> we can only verify what is the expected return value (which could be hard coded)